src/main/java/org/commcare/formplayer/session/MenuSession.java

```diff
@@ -385,17 +374,10 @@ public FormSession getFormEntrySession(FormSendCalloutHandler formSendCalloutHan
         FormDef formDef = engine.loadFormByXmlns(formXmlns);
         HashMap<String, String> sessionData = getSessionData();
         String postUrl = sessionWrapper.getPlatform().getPropertyManager().getSingularProperty("PostURL");
-        return new FormSession(sandbox, formDef, username, domain,
-                sessionData, postUrl, locale, uuid,
-                null, oneQuestionPerScreen,
-                asUser, appId, null, formSendCalloutHandler, storageFactory, false, null);
-    }
-
-    public void reloadSession(FormSession formSession) throws Exception {
```

Comment:
I presume this method was simply never used? I didn't see any other references in this PR.